src/handlers/Auth/OpenLoginHandler.js

```diff
@@ -76,34 +81,54 @@ class OpenLoginHandler {
     }
   }
 
+  async updateSession(sessionData) {
+    try {
+      const { sessionId } = this.openLoginInstance.state.store.getStore()
+      const { sessionNamespace } = this.openLoginInstance.state
+
+      if (sessionId) {
+        const privKey = Buffer.from(sessionId, 'hex')
+        const publicKeyHex = getPublic(privKey).toString('hex')
+        const encData = await encryptData(sessionId, sessionData)
+        const signatureBf = await sign(privKey, keccak256(encData))
+        const signature = signatureBf.toString('hex')
+        await post(`${config.storageServerUrl}/store/set`, { key: publicKeyHex, data: encData, signature, namespace: sessionNamespace })
```

Comment:
if we don't want to override pending sessionTime thn yes it should be a new api, however since i m sending all the data in this api rather thn only updated fields so it works fine.